src/main/java/io/anserini/collection/JsonCollection.java

```diff
@@ -164,10 +166,10 @@ public String contents() {
 
     @Override
     public String raw() {
-      if (contents == null) {
-        throw new RuntimeException("JSON document has no \"contents\" field");
+      if (raw == null) {
+        throw new RuntimeException("JSON document has no \"raw\" field");
```

Comment:
I don't think `raw` can ever be `null`? Can you check? If `raw` is `null`, then how did we read the document in the first place...?

Reply:
And either way, I don't think this is the right error message... "raw" isn't a field in the JSON...

Reply:
Thanks for pointing this out, `raw` can never be `null` as indexed docs always have `id`/`contents`.